administrator/components/com_admin/sql/updates/postgresql/5.0.0-2023-06-21.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "#__guidedtour_steps" ADD COLUMN "params" text NOT NULL, /** CAN FAIL **/;
```

Comment:
```suggestion
ALTER TABLE "#__guidedtour_steps" ADD COLUMN "params" text NOT NULL /** CAN FAIL **/;
```
Comma will cause syntax error.

Reply:
ARGH!! the perils of cut and paste :(